node/typings/Checkout.ts

```diff
@@ -371,6 +371,10 @@ interface RatesAndBenefitsData {
     name: string
     featured: boolean
     description: string
+    additionalInfo: {
+      key: string
+      value: string
+    }
```

Comment:
From the code that uses this field within `node/utils/simulation.ts` it seems like this type is wrong. It's defined as if this field always contained only these two key/value keys, for example: `{ key: "some key", value: "some value"}`. But from what I understood of this PR, this original field can actually have several keys, not only `key` and `value`, right? It's the resulting graphql response that follows this pattern, not the original data.

So I believe it should be something like this:

```suggestion
    additionalInfo?: Record<string, string>
```